src/components/screens/Cadastros/EditaisContratos/EditaisContratos.jsx

```diff
@@ -13,50 +16,53 @@ class EditaisContratos extends Component {
     super(props);
     this.state = {
       lotes: [],
-      lotesSelecionados: [
-        
-      ]
+      diretoriasRegionais: [],
+      empresas: [
+        {
+          label: 'SINGULAR GESTÃO DE SERVIÇOS LTDA',
+          value: '24B20725-17B1-4E83-8EE9-1B0CAD5C4F51'
+        },
+        {
+          label: 'APETECE SISTEMAS DE ALIMENTAÇÃO S/A.',
+          value: '366270C1-3156-4792-920C-399B1973C58D'
+        },
+        {
+          label: 'S.H.A COMÉRCIO DE ALIMENTOS LTDA',
+          value: 'F5CB749C-A1A8-4D4D-8276-AAD2EF3D9269'
+        },
+        {
+          label: 'P.R.M. SERVIÇOS E MÃO DE OBRA ESPECIALIZADA EIRELI ',
+          value: '3FA6EB36-7456-4274-824E-A4809494412A'
+        },
+        {
+          label: 'COMERCIAL MILANO BRASIL',
+          value: '097783FB-FEC6-4DD2-98EC-832138F294F9'
+        }
+      ],
+      forms: ["secaoEdital0"]
     };
-    this.lidarComLotesSelecionados = this.lidarComLotesSelecionados.bind(this);
+  }
+
+  nomeFormAtual() {
+    const nome = `secaoEdital${this.state.forms.length - 1}`;
```

Comment:
const indiceDoLALA = this.state.forms.length - 1

Reply:
const indiceDoFormAtual = `secaoEdital${this.state.forms.length - 1}`;